tools/unitctl/unit-openapi/README.md

```diff
@@ -409,3 +409,4 @@ cargo doc --open
 ## Author
 
 unit-owner@nginx.org
+
```

Comment:
Hmm, what happened here? It reintroduces (from diff --check)

```
tools/unitctl/unit-openapi/README.md:412: new blank line at EOF.
```

Reply:
If fixing it is problematic for some reason, we can always add a `tools/unitctl/unit-openapi/.gitattrbutes` file containing

```
README.md       whitespace=-blank-at-eof
```

Reply:
Ill add that for now, and a ticket to track fixing the generator.